changelog/deprecate_dualcontext.dd

```diff
@@ -0,0 +1,13 @@
+Deprecate local templates that receive local symbols by alias
+
+Support for local and member templates which can be instantiated with local
+symbols was added to the language in D 2.087.0, however there are a number of
+issues with the feature including:
+
+- Only DMD has code-gen support for the feature.
+- There has been no support to implement this feature in either GDC or LDC.
+- There is still no language specification that describes the feature.
+
+Because of this, there has been a dead feature introduced to the language for
+over nine releases, so now it has been deprecated, and due to be fully reverted
+in a future release.
```

Comment:
This wording is not OK and ought to be changed.

- The world does not revolve around GDC+LDC. As pointed out, many people use exclusively DMD, or are happy to use exclusively DMD in order to take advantage of its exclusive features.
- No language specification is needed for this feature because its absence is an implementation bug. The limitation is emergent from the current design of the DMD frontend.

Reply:
https://github.com/dlang/dmd/pull/12111